.github/workflows/test-build.yml

```diff
@@ -63,8 +77,21 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v4
         with:
+          fetch-depth: 0
           path: mdn/yari
 
+      - name: Merge main (if possible)
```

Comment:
Not so sure about merging main in automagically when we know there's going to be merge conflicts, or at least inconsistencies, while fred is in development